packages/libretro/mupen64plus/package.mk

```diff
@@ -64,8 +64,11 @@ make_target() {
     OdroidXU3)
       make platform=odroid BOARD=ODROID-XU3 GLES=1 FORCE_GLES=1 HAVE_NEON=1 WITH_DYNAREC=arm
       ;;
+    ROCK960)
```

Comment:
Why we need this?

Reply:
`ROCK960` case and `*` case are the same, why not remove the `ROCK960` case 
and let it fall through?

Reply:
exactly the reason i asked